static_src/stores/service_plan_store.js

```diff
@@ -54,7 +54,8 @@ export class ServicePlanStore extends BaseStore {
       case serviceActionTypes.SERVICE_PLAN_RECEIVED: {
         const servicePlan = action.servicePlan;
         const servicePlanReceived = Object.assign({}, servicePlan, { fetching: false });
-        this.merge('guid', servicePlanReceived);
+        this.merge('guid', servicePlanReceived, () => { });
+        this.emitChange();
```

Comment:
This emit change shouldn't be necessary, if the `servicePlanReceived` was changed then `merge` will emit for us. We're not modifying any other state here outside of the model.